test/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "bedrock-account-http-test",
+  "version": "0.0.1-0",
+  "description": "Bedrock key-http test",
+  "private": true,
+  "main": "test",
+  "scripts": {
+    "test": "node --preserve-symlinks test.js test",
+    "watch": "grunt watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/bedrock-account-http"
+  },
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-account-http/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-account-http",
+  "dependencies": {
+    "apisauce": "^1.0.2",
+    "bedrock": "^1.12.1",
+    "bedrock-account": "^2.1.0",
+    "bedrock-account-http": "file:..",
+    "bedrock-did-client": "^1.1.2",
+    "bedrock-docs": "^2.0.1",
+    "bedrock-express": "^2.0.6",
+    "bedrock-identity": "^6.1.0",
+    "bedrock-key": "^5.1.2",
+    "bedrock-mongodb": "^5.0.0",
+    "bedrock-passport": "^4.0.4",
+    "bedrock-permission": "^2.4.2",
+    "bedrock-rest": "^2.0.0",
+    "bedrock-server": "^2.1.3",
+    "bedrock-test": "^2.0.0",
+    "bedrock-validation": "^4.0.0",
+    "bedrock-views": "^5.4.1",
+    "eslint-config-digitalbazaar": "^1.1.0",
```

Comment:
this is not required here is it?  It should be getting installed at the root level?